vspace/src/main/java/edu/asu/diging/vspace/web/staff/SpaceController.java

```diff
@@ -38,6 +38,8 @@ public String showSpace(@PathVariable String id, Model model) {
     	
     	
         ISpace space = spaceManager.getFullyLoadedSpace(id);
+        model.addAttribute("linksOnThisSpace", spaceManager.getLinkedSpaces(id));
+        model.addAttribute("linksToThisSpace",spaceManager.getLinkedFromSpaces(id));
```

Comment:
method names are confusing, why not `getLinksToSpaces` and `getLinksFromSpaces`? Or even better 'getOutgoingLinks` and 'getIncomingLinks`?